tests/tracer/runtime/test_container.py

```diff
@@ -213,6 +225,7 @@ def test_cgroup_info_from_line(line, expected_info):
 3:blkio:/ecs/55091c13-b8cf-4801-b527-f4601742204d/432624d2150b349fe35ba397284dea788c2bf66b885d14dfc1569b01890ca7da
 2:memory:/ecs/55091c13-b8cf-4801-b527-f4601742204d/432624d2150b349fe35ba397284dea788c2bf66b885d14dfc1569b01890ca7da
 1:name=systemd:/ecs/55091c13-b8cf-4801-b527-f4601742204d/432624d2150b349fe35ba397284dea788c2bf66b885d14dfc1569b01890ca7da
+1:name=systemd:/ecs/34dc0b5e626f2c5c4c5170e34b10e765-1234567890
```

Comment:
Why do we have this here? The previous line is always going to get picked up right? Does this prove we do them in that order? Should we have a separate `Fargate 1.4 file` example we test with?